package.json

```diff
@@ -1969,6 +1978,7 @@
     "vscode-cpptools": "^5.0.0",
     "vscode-extension-telemetry": "^0.1.6",
     "vscode-nls": "^5.0.0",
+    "webpack": "^5.38.1",
```

Comment:
I think webpack is supposed to stay in `devDependencies`. cpptools has it there too.